zcash_client_backend/src/fees.rs

```diff
@@ -27,28 +27,44 @@ pub struct ChangeValue {
     output_pool: PoolType,
     value: NonNegativeAmount,
     memo: Option<MemoBytes>,
+    is_ephemeral: bool,
```

Comment:
The addition of this boolean suggests that `ChangeValue` should actually be an enum (at least as far as the Rust type is concerned; parsing could handle incorrect configurations):
```
pub enum ChangeValue {
    Shielded { protocol: ShieldedProtocol, value: NonNegativeAmount, memo: Option<MemoBytes> },
    EphemeralTransparent { value: NonNegativeAmount }
}
```